CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKIconsRenderable.java

```diff
@@ -1163,11 +1160,16 @@ public int CreateCommandBuffers(){
         int imageCount = cvkSwapChain.GetImageCount();
         
         commandBuffers = new ArrayList<>(imageCount);
-
+        offscreenCommandBuffers = new ArrayList<>(imageCount);
```

Comment:
Can it use the same ones again?

Reply:
no cos they are being recorded in by the main render code

Reply:
Actually I wonder if there is a way to record the display render and hittest render into the same commandbuffer?  Anyway problem for another change.

Reply:
Hmm possibly, all the examples I have looked at (to be far there aren't many) use separate buffers. Happy to have a look into it though. My plan is to get it working with the known method then look into potentially better ways